sw/device/silicon_creator/lib/ownership/owner_block.c

```diff
@@ -329,6 +344,16 @@ rom_error_t owner_block_flash_check(const owner_flash_config_t *flash) {
   return kErrorOk;
 }
 
+rom_error_t owner_isfb_config_check(const owner_isfb_config_t *isfb) {
+  if (is_owner_page(isfb->bank, isfb->page) == kHardenedBoolFalse) {
```

Comment:
We should also check the `isfb->header.length` here, just to make sure the TLV struct is the correct size before we start looking at the rest of the fields.

Reply:
Fixed.